crates/project/src/project.rs

```diff
@@ -9523,7 +9523,7 @@ impl ProjectLspAdapterDelegate {
                 format!("failed to determine load login shell environment in {worktree_abs_path:?}")
             })
             .log_err()
-            .unwrap_or_default();
+            .unwrap_or_else(|| HashMap::from_iter(std::env::vars()));
```

Comment:
I don't actually know if this is fully necessary, but I figured this was a good default instead of the hash map being empty. Thoughts?

Reply:
If it’s not necessary then I think it may be best to leave it as-is.

Reply:
Yeah, let's leave it empty unless there's a concrete use case for it.

Reply:
Reverted! At the very least `clangd` worked just fine without env vars set. If it breaks some other language server on Windows I'm sure it'll be reported :P